index.bs

```diff
@@ -1719,6 +1719,45 @@ that should be considered in the development of new features, notably:
 * [[#feature-detect]]
 * Polyfill development should be encouraged
 
+<h3>Extend existing manifest files rather than creating new ones</h3>
+
+Manifest files are sometimes required. However, they can also add complexity to the web platform,
+which creates developer confusion. The use of different manifests on the web should therefore be kept to a minimum.
+
+Metadata can reside inside a new manifest if that is related to a different function or the fetch timing 
+of that metadata is different. Application metadata should be added to the Web App Manifest or be an extension of it. 
+Parameters for a specific function (like Payment Method Manifest) are a completely different type. Publication Manifest,
```

Comment:
This sentence (starting "Parameters for a specific function" is extremely hard to understand.

Reply:
I think this came from @dbaron but might be wrong. I am unsure about what else to write or how to rephrase it